packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -245,9 +245,7 @@ class ReorderableList extends StatefulWidget {
   final DragStartBehavior dragStartBehavior;
 
   /// {@macro flutter.widgets.scroll_view.keyboardDismissBehavior}
```

Comment:
This template doc should be updated to reflect what the default behavior is, and how it is provided by the ScrollBehavior.

Reply:
I've pushed a new version where the `{@template flutter.widgets.scroll_view.keyboardDismissBehavior}` is updated to mention the logic behind it (how the value fallback).